src/kubernetes_api/core.clj

```diff
@@ -45,9 +74,10 @@
   (client \"https://kubernetes.docker.internal:6443\"
            {:basic-auth {:username \"admin\"
                          :password \"1234\"}
-            :openapi {:apis [\"some.api/v1alpha1\", \"another.api\"]}})"
+            :apis [:some.api/v1alpha1, :another.api/v1beta1]})"
```

Comment:
I moved the apis outside the `:openapi`, since it's not about just the schema, but the whole client will only work for these API versions